packages/ui/src/Integration/IntegrationDetailEditableName.tsx

```diff
@@ -0,0 +1,23 @@
+import { Icon } from 'patternfly-react';
+import * as React from 'react';
+
+export interface IIntegrationDetailEditableNameProps {
+  name?: string;
+}
+
+export class IntegrationDetailEditableName extends React.PureComponent<
```

Comment:
I do feel this component may be overkill.. but it will be used across the three tab pages, now that they will be made as pages.